flowmachine/flowmachine/core/server/query_schemas/subscriber_degree.py

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marshmallow import Schema, fields, post_load
+from marshmallow.validate import OneOf, Length
+
+from flowmachine.features import SubscriberDegree
+from .base_exposed_query import BaseExposedQuery
+from .custom_fields import SubscriberSubset
+
+__all__ = ["SubscriberDegreeSchema", "SubscriberDegreeExposed"]
+
+
+class SubscriberDegreeSchema(Schema):
+    query_kind = fields.String(validate=OneOf(["subscriber_degree"]))
+    start = fields.Date(required=True)
+    stop = fields.Date(required=True)
+    direction = fields.String(
+        required=False, validate=OneOf(["in", "out", "both"]), default="both"
+    )  # TODO: use a globally defined enum for this
+    subscriber_subset = SubscriberSubset()
+
+    @post_load
+    def make_query_object(self, params):
```

Comment:
As of #990, this needs a `**kwargs` argument due to the dependency upgrade of marshmallow to `3.0.0.rc7` which [introduced](https://marshmallow.readthedocs.io/en/3.0/upgrading.html#decorated-methods-receive-many-and-partial) a backwards-incompatible change in methods decorated with `@post_load`.

Reply:
(This is what's causing the currently failing integration test on CircleCI.)

Reply:
That makes sense. I will change it now.